typescript/infra/config/environments/mainnet3/warp/EZETH-deployments.yaml

```diff
@@ -9,7 +9,7 @@ data:
       name: Renzo Restaked ETH
       symbol: ezETH
       hypAddress: '0xC59336D8edDa9722B4f1Ec104007191Ec16f7087'
-      tokenAddress: '0x2416092f143378750bb29b79eD961ab195CcEea5'
```

Comment:
Update the token so that we use the token address of EZETH on ethereum, this address was not being used before